tox.ini

```diff
@@ -22,9 +29,12 @@ commands =
     pip install -e .
     py.test {posargs}
```

Comment:
If this is also making a few amends to the tox file could change this to just `pytest`?

I also wonder about adding `-W error` so any deprecation warnings get caught earlier?

Reply:
I changed it to `pytest`.

> I also wonder about adding -W error so any deprecation warnings get caught earlier?

I am not sure about this TBH.